README.md

```diff
@@ -67,23 +67,39 @@ $ $GOPATH/bin/terraform-provider-ionoscloud
 ...
 ```
 
+## Testing the Provider
+
+⚠️ **Warning:** Acceptance tests create real resources, and often cost money to run.
+
 In order to test the provider, you can simply run `make test`.
 
-```sh
+``` bash
 $ make test
 ```
 
 In order to run the full suite of Acceptance tests, run `make testacc TAGS=all`.
 
-Tests can be split using build tags: all(all tests), compute(all compute-engine tests), dbaas, k8s, server, lan, ipblock.
+``` bash
+$ make testacc TAGS=all
+```
 
-⚠️ **_Note: Acceptance tests create real resources, and often cost money to run._**
+### Test Tags
 
-```sh
-$ make testacc TAGS=all
+If you want to test a specific batch of tests, they are split by **build tags** as follows:
+
+- `compute` - all **compute engine** tests (datacenter, firewall rule, image, IP block, IP failover, lan, location, nic, private cross connect, server, snapshot, template, volume)
+- `nlb` - **network load balancer network load balancer forwarding rule tests**
```

Comment:
it seems like *network load balancer* is duplicated here, maybe there is a missing **and**?